test/Microsoft.VisualStudio.Validation.Tests/AssumesTests.cs

```diff
@@ -79,28 +79,40 @@ public void Fail()
     [Fact]
     public void NotNull()
     {
-        Assert.ThrowsAny<Exception>(() => Assumes.NotNull((object?)null));
+        Exception ex = Assert.ThrowsAny<Exception>(() => Assumes.NotNull((string?)null));
+
+        Assert.Equal("Unexpected null value of type 'String'.", ex.Message);
```

Comment:
This will cause tests to fail when built on non en-US agents, which has bitten us before, as this is an OSS project and other folks can run tests. 
Instead, can you use `Assert.Contains` here and just test that the type name appears somewhere in the string?

Reply:
Back from vacation and picking this up. Would it be enough to set `CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;` in the test fixture in order to have a more robust assertion that still works when executed in different cultures?

Reply:
Yes, that would be fine.

Reply:
Update pushed.